helpers/commonFunc.js

```diff
@@ -1,11 +1,7 @@
-const genToken = ({ id, email, expiresIn = false }) => {
+const genToken = ({ id, name, email, expiresIn }) => {
```

Comment:
why need this destructurization?
why need name param?
if i want send another param?